Resources/Prototypes/Magic/event_spells.yml

```diff
@@ -10,3 +10,195 @@
+        orGroup: Guns
+      - id: WeaponShotgunDoubleBarreled
+        orGroup: Guns
+      - id: WeaponShotgunEnforcer
+        orGroup: Guns
+      - id: WeaponShotgunKammerer
+        orGroup: Guns
+      - id: WeaponShotgunSawn
+        orGroup: Guns
+      - id: WeaponShotgunHandmade
+        orGroup: Guns
+      - id: WeaponShotgunBlunderbuss
+        orGroup: Guns
+      - id: WeaponShotgunImprovised
+        orGroup: Guns
+      - id: WeaponSubMachineGunAtreides
+        orGroup: Guns
+      - id: WeaponSubMachineGunC20r
+        orGroup: Guns
+      - id: WeaponSubMachineGunDrozd
+        orGroup: Guns
+      - id: WeaponSubMachineGunWt550
+        orGroup: Guns
+      - id: WeaponSniperMosin
+        orGroup: Guns
+      - id: WeaponSniperHristov
+        orGroup: Guns
+      - id: Musket
+        orGroup: Guns
+      - id: WeaponPistolFlintlock
+        orGroup: Guns
+      - id: WeaponLauncherChinaLake
+        orGroup: Guns
+      - id: WeaponLauncherRocket
+        orGroup: Guns
+      - id: WeaponLauncherPirateCannon
+        orGroup: Guns
+      - id: WeaponTetherGun
+        orGroup: Guns
+      - id: WeaponForceGun
+        orGroup: Guns
+      - id: WeaponGrapplingGun
+        orGroup: Guns
+      - id: WeaponLightMachineGunL6
+        orGroup: Guns
+      - id: WeaponLaserSvalinn
+        orGroup: Guns
+      - id: WeaponLaserGun
+        orGroup: Guns
+      - id: WeaponMakeshiftLaser
+        orGroup: Guns
+      - id: WeaponTeslaGun
+        orGroup: Guns
+      - id: WeaponLaserCarbinePractice
+        orGroup: Guns
+      - id: WeaponLaserCarbine
+        orGroup: Guns
+      - id: WeaponPulsePistol
+        orGroup: Guns
+      - id: WeaponPulseCarbine
+        orGroup: Guns
+      - id: WeaponPulseRifle
+        orGroup: Guns
+      - id: WeaponLaserCannon
+        orGroup: Guns
+      - id: WeaponParticleDecelerator
+        orGroup: Guns
+      - id: WeaponXrayCannon
+        orGroup: Guns
+      - id: WeaponDisablerPractice
+        orGroup: Guns
+      - id: WeaponDisabler
+        orGroup: Guns
+      - id: WeaponDisablerSMG
+        orGroup: Guns
+      - id: WeaponTaser
+        orGroup: Guns
+      - id: WeaponAntiqueLaser
+        orGroup: Guns
+      - id: WeaponAdvancedLaser
+        orGroup: Guns
+      - id: WeaponPistolCHIMP
+        orGroup: Guns
+      - id: WeaponBehonkerLaser
+        orGroup: Guns
+      - id: WeaponEnergyShotgun
+        orGroup: Guns
+      - id: WeaponMinigun
+        orGroup: Guns
+      - id: BowImprovised
+        orGroup: Guns
+      - id: WeaponFlareGun
+        orGroup: Guns
+      - id: WeaponImprovisedPneumaticCannon
+        orGroup: Guns
+      - id: WeaponWaterPistol
+        orGroup: Guns
+      - id: WeaponWaterBlaster
+        orGroup: Guns
+      - id: WeaponWaterBlasterSuper
+        orGroup: Guns
+      - id: RevolverCapGun
+        orGroup: Guns
+      - id: RevolverCapGunFake
+        orGroup: Guns
+      speech: action-speech-spell-summon-guns
```

Comment:
These guns should probably have different probabilities.
We don't want several captain's lasers and energy shotguns around each time it is used on a high pop server.
But the exact balance can be adjusted later